target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz3390.dts

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz3390", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 3390";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_red;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_red;
+
+		led-dsl = &led_dsl;
+		led-internet = &led_info;
+		led-wifi = &led_wifi;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		power {
+			label = "power";
+			gpios = <&gpio 1 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_POWER>;
+		};
+
+		wifi {
+			label = "wifi";
+			gpios = <&gpio 29 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_RFKILL>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_green: power_green {
+			label = "green:power";
+			gpios = <&gpio 45 GPIO_ACTIVE_LOW>;
+			default-state = "keep";
+		};
+
+		led_power_red: power_red {
+			label = "red:power";
+			gpios = <&gpio 46 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wifi: wifi {
+			label = "green:wifi";
+			gpios = <&gpio 36 GPIO_ACTIVE_LOW>;
+		};
+
+		led_dsl: dsl {
+			label = "green:dsl";
+			gpios = <&gpio 35 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lan {
+			label = "green:lan";
+			gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
+		};
+
+		led_info: info {
+			label = "green:info";
+			gpios = <&gpio 33 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-export {
+		compatible = "gpio-export";
+
+		gpio_wasp_reset {
+			gpio-export,name = "fritz3390:wasp:reset";
+			gpio-export,output = <1>;
+			gpios = <&gpio 34 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
```

Comment:
This export is only required if you bring up the secondary WiFi SoC. Personally, I would remove it for now - please wait for other reviewer's comments before doing so.

Reply:
Okay, we should decide about this.

Reply:
Also I woud prefer to remove this node.